src/annotation-engine/stories.tsx

```diff
@@ -9,8 +9,12 @@ export default {
     component: AnnotationEngine,
     argTypes: {
         onAnnotationEnded: { action: 'Annotation ended' },
+        drawingEvent: {
+            control: { type: 'select', options: ['drag', 'mousemove'], }
```

Comment:
Si on suit le commentaire au dessus 
```suggestion
            control: { type: 'select', options: Object.values(DrawingEvent) }
```